apps/demos/jest.config.js

```diff
@@ -1,5 +1,4 @@
 module.exports = {
-  testEnvironment: 'node',
```

Comment:
In new `jest`, the default `testEnvironment` is `node` (used to be `dom`). So, now I removed every `node` value and added `dom` values where required